enodi/post_it
# #157032342 Implement landing page

#### What does this PR do?
This PR implements the application landing page

#### Description of Task to be completed?
- Ensures user sees landing page when a user navigates to the application.

#### How should this be manually tested?
- Pull this branch and run `foreman start -f Procfile.dev -p 3000`
- Navigate to `localhost:300`. You'll see the landing page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: 157032342](https://www.pivotaltracker.com/n/projects/2166405/stories/157032342)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/39576927-bed0469c-4ed7-11e8-90c2-9248d0f042b6.png)
